### DISCUSSION TITLE
Add simulated time props to rux-clock

### DISCUSSION THREAD

**Discussion by kit047** [RocketCommunicationsInc/astro#936] [Feature Requests]
rux-clock is always ticking. It would be nice if the clock could be used as to display simulated time as well. The `date-in` prop is nice to set the time of the clock, but the clock continues to tick afterwards. It would be nice be able to set the time with `date-in` and then control when the clock ticks, either by a `play/pause` prop or by continuously setting the `date-in` prop. 

A use-case for this is when running a simulation at a certain datetime, having the clock reflect the current time in the simulation even when the simulation is paused.

**Comment by markacianfrani** [comment]
Thanks for raising this! I'm working on a proposal for a new, more generic clock segment component that should fit this use case nicely. The current Clock is composed of several clock-like atoms (a datetime + label), but those pieces aren't flexible enough. This new component would behave more like the AOS/LOS timestamps and would allow you to control the time yourself. 

I'll follow up as soon as I have more details regarding status/priority/timeline

**Comment by markacianfrani** [comment]
we've added a new [`static` attribute](https://github.com/RocketCommunicationsInc/astro/pull/989) to clock to disable the ticking feature. with this, you should now be able to control the time yourself via the `date-in` attribute. you may need to write some of your boilerplate code to handle play/pause yourself, but this will give you the most flexibility (for example, you can also do things like rewind, fast foward now) this feature will be available in our next release (2022-12-22)

**Comment by markacianfrani** [comment]
we've just released 7.5.0 which includes the new static attribute to clock
